app/views/posts/index.html.erb

```diff
@@ -1 +1,34 @@
-<h1>users posts</h1>
\ No newline at end of file
+<main>
+    <div class="user-container">
+        <section class="user">
+            <img src=<%= @user.photo %> alt="User Photo" class="user-photo">
+            <div class="user-details">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <h5 class="user-post-count">Number of posts: <%= @user.posts_counter %> </h5>
+            </div>
+        </section>
+        <section class="user-posts">
+            <h3>All Posts from <%= @user.name %></h3>
+            <% @user.posts.each do |post| %>
+            <a href="/users/<%=@user.id%>/posts/<%=post.id%>" class="user-post-link">
+                <div class="post">
+                    <h4><%= post.title %></h4>
+                    <p><%= truncate(post.text, length: 180) %></p>
+                    <p class="comment-like-count">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+                </div>
+                <div class="comments">
+                    <h4>Comments</h4>
+                    <% post.five_latest_comments.each do |comment| %>
+                        <p  id="single-comment">
+                            <%= comment.author.name %> : <%= comment.text %>
+                        </p>
+                    <% end %>
+                </div>
+                <button class="button">
+                  <a href="/users/<%=@user.id%>/posts/<%=post.id%>" class="user-post-link">Pagination</a>
+                </button>
+            </a>
+            <% end %>
+        </section>
```

Comment:
Good job so far. However, kindly check the following: 

- [ ] The pagination button is suppose to be for the whole page and not individual post card as done above.
- [ ] You are suppose to display the post number ( eg. Post `#1`) as shown in the design.
Kindly work on the above and update your project accordingly. 


### Your design 

![Screenshot from 2023-06-24 21-19-35](https://github.com/AlaaAlsalem/blogApp/assets/65233860/574e340f-97cb-4991-ba77-05b154b8f516)



### Actual design 

![Screenshot from 2023-06-24 21-20-19](https://github.com/AlaaAlsalem/blogApp/assets/65233860/d2df6f31-50d8-4d93-840c-910d1b3ba2ae)